src/metabase/driver/postgres.clj

```diff
@@ -220,13 +221,15 @@
   [driver [_ arg]]
   (sql.qp/->honeysql driver [:percentile arg 0.5]))
 
+(p/defrecord+ RegexMatchFirst [identifier pattern]
```

Comment:
possible nit: why PascalCasing on `RegexMatchFirst`?

Reply:
It's a record type so `RegexMatchFirst` is a class. Normal Clojure convention is to use pascal case for classes https://guide.clojure.style/#naming-protocols-records-structs-and-types